src/components/formio/radio.tsx

```diff
@@ -65,44 +67,55 @@ export const Radio: React.FC<RadioProps> = ({
   isClearable = false,
   tooltip = '',
   description = '',
+  isLoading = false,
 }) => {
+  const intl = useIntl();
   const {getFieldProps, setFieldValue} = useFormikContext<ExtendedComponentSchema>();
   const {value} = getFieldProps(name);
   const hasSelection = !!options.find(opt => opt.value === value);
 
   return (
-    <Component type="radio" field={name} label={label} tooltip={tooltip} required={required}>
-      <div className="form-radio radio">
-        {options.map(({value, label, description}, index) => (
-          <div key={`option-${value}-${index}`} className="form-check">
-            <label className="form-check-label">
-              <RadioInput
-                name={name}
-                value={value}
-                label={label || <EmptyLabel />}
-                description={description}
-              />
-            </label>
+    <>
+      {isLoading ? (
+        intl.formatMessage({
```

Comment:
please use `<FormattedMessage ... />` if you can render normal React components. `intl.formatMessage` is a fallback for when you can't return a react component.